docs/how-to/websites-on-ipfs/redirects-file-support.md

```diff
@@ -0,0 +1,96 @@
+---
+title: redirects file support
+description: What the _redirect file is and how to use them with a website on IPFS.
+---
+# `_redirects` file support
```

Comment:
```suggestion
# Redirects file support
```

Try to keep formatting, like code blocks or bolding, out of titles. Also, use sentence case for titles.